tests/simple_ubo/simple.cpp

```diff
@@ -71,7 +71,7 @@ int main(int argc, char* argv[])
     CHECK_CL_ERRCODE(err);
 
     // Build program
-    err = clBuildProgram(program, 1, &device, " -constant-args-ubo -inline-entry-points ", nullptr, nullptr);
+    err = clBuildProgram(program, 1, &device, " -constant-args-ubo -inline-entry-points -relaxed-ubo-layout ", nullptr, nullptr);
```

Comment:
`-relaxed-ubo-layout` isn't really needed, is it?

Reply:
No that was in there from an earlier debugging effort. I forgot to remove it.

Reply:
Thanks for confirming.